welcome/install-d-locally.md

```diff
@@ -0,0 +1,50 @@
+# Instalar o D localmente
+
+O compilador de referência da linguagem D chama-se DMD (Digital Mars D).
+Existem outros compiladores, sendo os mais relevantes o
+[LDC](https://github.com/ldc-developers/ldc) (compilador para D basedado na
+[LLVM](http://llvm.org)) e o [GDC](https://gdcproject.org) (compilador para D
+baseado no [GCC](https://gcc.gnu.org/)).
+Informação mais detalhada está disponível na
+[página da wiki sobre Compiladores](https://wiki.dlang.org/Compilers), mas para
+pessoas novas à linguagem é recomendado instalar o DMD.
+
+## Download e Instalação
+
+A [página de downloads do D](https://dlang.org/download.html) tem informação
+sobre as várias implementações da linguagem e contém links para _packages_
+pré-compilados e específicos aos vários sistemas operativos prontos para fazer
+o dowload e instalar.
+
+Como alternativa aos _packages_ específicos ao sistema operativo, também está
+disponivel um [_script_ de instalação](https://dlang.org/install.html) para
+qualquer sistema baseado em Posix (Linux, FreeBSD, MacOS) que pode instalar
+várias implementações diferentes (incluindo várias versões) localmente sem
+ser necessário permissões de administrador.
+Informação mais detalhada encontra-se na
+[documentação do _script_ de instalação](https://dlang.org/install.html).
+
+## Conguração do editor
+
+Para usar o D não é necessário nenhum IDE muito sofisticado, o que é muito raro.
+Contudo, programar em D é melhor quando usamos o nosso editor favorito.Existem
```

Comment:
`...nosso editor favorito. Existem plugins...` (add a space in `favorito.Existem`)